### PR TITLE
feat: add inohara as a default client

### PR DESCRIPTION
**Pull request by nyairobi**
https://gitea.tendokyu.moe/akanyan/inohara

### CODE REVIEW

File: server/src/lib/builtin-clients/builtin-clients.ts [hunk 1]
```diff
@@ -223,6 +223,47 @@ import = '/ir/direct-manual/import'
...
+		apiKeyTemplate: `[Options]
+
+# Whether to enable score submissions
+Enable = true
+
+# Timeout for web requests, in seconds
+Timeout = 3
+
+# Tachi instance base URL
+BaseUrl = ${ServerConfig.OUR_URL}
```
Comment (zkldi) [inline server/src/lib/builtin-clients/builtin-clients.ts:242]: does this really work without quotes around the url?

**Review by zkldi — APPROVED** [review]

File: server/src/lib/builtin-clients/builtin-clients.ts (hunk 1 above)
Comment (nyairobi) [reply server/src/lib/builtin-clients/builtin-clients.ts:242]: Why wouldn't it? This is not TOML

Comment (zkldi) [reply server/src/lib/builtin-clients/builtin-clients.ts:242]: it's very rare that i see a config format correctly parse unquoted strings

Comment (nyairobi) [reply server/src/lib/builtin-clients/builtin-clients.ts:242]: And I think I've never seen an ini dialect that would complain about unquoted values.
You can find the parser n Export.cs:58. It's very basic.

**Pull request merged**